m2c_helper.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+#
+# m2c_helper.sh
+# Utility to aid in the generation of m2c output for specific functions
+#
+# example usages:
+#  m2c_helper.sh code_80091750 func_800AAC18
+#  m2c_helper.sh audio/load audio_init
+#
+# Be sure to set DISASEMBLY_FILE to a filename you are comfortable with.
+#   Defaulted to function_disassembly.c
+#
+# Its expected that the utiltity will be ran twice for any given function.
+# The first run will create a context file specific to the target function
+# The second run will create the full disassembly input,
```

Comment:
I think this isn't worded the best as I find it difficult to understand.

What about?
`The second run outputs the full m2c`

Reply:
Oh, yeah, I messed that sentence up badly